wonderitems.txt

```diff
@@ -0,0 +1,84 @@
+38767036: (0, 7, 0, 9, 'Inside the Deku Tree', {'type': 3, 'type_string': 'INTERACT_SWITCH', 'drop': 'HEART', 'damage_type': 0})
```

Comment:
Can this file be moved into `Notes`?

Reply:
I just deleted it, this was just the output of the script I mentioned above that I used to find all of the wonderitems.